git-whatchanged: add page (#5935)
tldr-pages/tldr: pages/common/git-whatchanged.md

# git whatchanged

> > Show what has changed with recent commits or files.

Reviewer: ```suggestion
> Show commit logs and diff outputs for recent commits or files.
```

> > See also `git log`.

Reviewer: ```suggestion
> See also `git log` for human readable logs.
```

> > More information <https://git-scm.com/docs/git-whatchanged>.

Reviewer 2: ```suggestion
> More information: <https://git-scm.com/docs/git-whatchanged>.
```

> - Display log of recent commits:

Reviewer: ```suggestion
- Display commit logs and diff outputs for recent commits:
```

Author: well thats just the man page, I thought tldr aimed to shorten and simplify man pages, not copying content from them

`git whatchanged`

> - Display log of recent commits within the specified time frame:

Reviewer: ```suggestion
- Display commit logs and diff outputs for commits within the specified time frame:
```

> `git whatchanged --since="2 hours ago"`

Reviewer 2: ```suggestion
`git whatchanged --since="{{2 hours ago}}"`
```

Author: Not sure about this

Reviewer 2: We normally do that whenever the thing is user-configurable (e.g. a variable or literal).  It also helps distinguish flags from values, etc., as @marchersimon pointed out [here](https://github.com/tldr-pages/tldr/pull/5921#issuecomment-836462426).

- Display changes to selected file(s):

> `git what-changed {{file}}`

Author: I agree with the specified part however I don't feel you need to have path

It needs to be something like 

{{./path/to/file/or/folder}}
the dot is needed because you are already in a git repo and git wouldn't work without it since it would give 
fatal: not a git repository (or any of the parent directories): .git

seems easier to let the user figure out they need to cd into the repo 

can you shorten this "{{./path/to/file/or/folder}}"

> `git what-changed {{file}}`

Reviewer 2: I'm not sure what you mean @CleanMachine1.  The `./` isn't needed — I just tried it and it worked fine with `path/to/file`.

> `git what-changed {{file}}`

Reviewer 2: We always do `path/to/file`, unless (1) there's a lot of args, or (2) a simple relative path wouldn't work (3) only a filename (without directories) can be used.

> `git what-changed {{file}}`

Author: not having the / before should work,
so 
either 

./path/to/directory/or/file
or 
path/to/directory/or/file
thats my thoughts on it

it shouldn't have a / to begin since that will start in the root directory /

> `git what-changed {{file}}`

Author: added in commit
